analyses/gistic-cohort-vs-histology-comparison/gistic-cohort-vs-histology-comparison.Rmd

```diff
@@ -0,0 +1,342 @@
+cohort_gistic_zip <- file.path(data_dir, "pbta-cnv-consensus-gistic.zip")
+
+if (!dir.exists(cohort_gistic_dir)) {
+  unzip(cohort_gistic_zip, 
+        exdir = data_dir)
+}
+
+lgat_gistic_dir <- file.path(data_dir, "pbta-cnv-consensus-lgat-gistic")
+lgat_gistic_zip <- file.path(data_dir, "pbta-cnv-consensus-lgat-gistic.zip")
+
+if (!dir.exists(lgat_gistic_dir)) {
+  unzip(lgat_gistic_zip, 
+        exdir = data_dir)
+}
+
+hgat_gistic_dir <- file.path(data_dir, "pbta-cnv-consensus-hgat-gistic")
+hgat_gistic_zip <- file.path(data_dir, "pbta-cnv-consensus-hgat-gistic.zip")
+
+if (!dir.exists(hgat_gistic_dir)) {
+  unzip(hgat_gistic_zip, 
+        exdir = data_dir)
+}
+
+medulloblastoma_gistic_dir <- file.path(data_dir, "pbta-cnv-consensus-medulloblastoma-gistic")
+medulloblastoma_gistic_zip <- file.path(data_dir, "pbta-cnv-consensus-medulloblastoma-gistic.zip")
+
+if (!dir.exists(medulloblastoma_gistic_dir)) {
+  unzip(medulloblastoma_gistic_zip, 
+        exdir = data_dir)
+}
+
+```
+
+### Run `plot_gistic_scores` for each file
+
+```{r}
+# Define file paths to `scores.gistic` files and run `plot_gistic_scores`
+# function for each file
+cohort_scores <-
+  file.path(cohort_gistic_dir, "scores.gistic")
+cohort_scores_plot <- plot_gistic_scores(cohort_scores)
+
+lgat_scores <-
+  file.path(lgat_gistic_dir, "scores.gistic")
+lgat_scores_plot <- plot_gistic_scores(lgat_scores)
+
+hgat_scores <-
+  file.path(hgat_gistic_dir, "scores.gistic")
+hgat_scores_plot <- plot_gistic_scores(hgat_scores)
+
+medulloblastoma_scores <-
+  file.path(medulloblastoma_gistic_dir, "scores.gistic")
+medulloblastoma_scores_plot <-
+  plot_gistic_scores(medulloblastoma_scores)
+```
+
+### Generate multipanel plots 
+
+```{r fig.height = 10, fig.width = 24}
+lgat_plot_list <- list(lgat_scores_plot, cohort_scores_plot)
+generate_multipanel_plot(plot_list = lgat_plot_list,
+                         plot_title = "LGAT scores versus entire cohort",
+                         output_directory = plots_dir,
+                         output_filename = "lgat_gistic_scores_multipanel_plot.png")
+
+hgat_plot_list <- list(hgat_scores_plot, cohort_scores_plot)
+generate_multipanel_plot(plot_list = hgat_plot_list,
+                         plot_title = "HGAT scores versus entire cohort",
+                         output_directory = plots_dir,
+                         output_filename = "hgat_gistic_scores_multipanel_plot.png")
+
+medulloblastoma_plot_list <- list(medulloblastoma_scores_plot, cohort_scores_plot)
+generate_multipanel_plot(plot_list = medulloblastoma_plot_list,
+                         plot_title = "Medulloblastoma scores versus entire cohort",
+                         output_directory = plots_dir,
+                         output_filename = "medulloblastoma_gistic_scores_multipanel_plot.png")
+```
+
+## Compare `amp_genes.conf_90.txt` and `del_genes.conf_90.txt` files
+
+### Custom function
+```{r}
+plot_amp_del_genes <-
```

Comment:
I think what is being compared in the Venn diagram is entire strings? Imagine a situation where the wide peaks between the histology-specific run and the entire cohort GISTIC run have a lot of overlap but are not exactly the same. It could be the case that 70% of the genes from the wide peaks from either run would overlap but show up as a zero in the the Venn diagram. We would want the 70% of genes in both to be reflected in the Venn diagram. So what we're after here is looking at the overlap between vectors of gene symbols, where the vector will contain the unique set of gene symbols that are in _any_ of the significant wide peaks. Does that make sense?

Reply:
Yes, that does make sense.